bigbluebutton-client/src/org/bigbluebutton/modules/screenshare/managers/WebRTCDeskshareManager.as

```diff
@@ -166,7 +167,7 @@ package org.bigbluebutton.modules.screenshare.managers
 			 sends out fall back to java command
 		*/
 		private function cannotUseWebRTC (message:String):void {
-			LOGGER.debug("WebRTCDeskshareManager::handleStartSharingEvent - falling back to java");
+			JSLog.warn("Cannot use WebRTC Screensharing: ", message);
```

Comment:
Have you considered duplicating the log so one copy would be LOGGER and the other would be JSLog? (same for `webRTCWorksButNotConfigured`)